action/protocol/execution/evm/evm.go

```diff
@@ -365,8 +366,10 @@ func executeInEVM(ctx context.Context, evmParams *Params, stateDB *StateDBAdapte
 	var (
 		contractRawAddress = action.EmptyAddress
 		executor           = vm.AccountRef(evmParams.txCtx.Origin)
+		london             = evm.ChainConfig().IsLondon(evm.Context.BlockNumber)
```

Comment:
isLondon

Reply:
`london` is used in our go-ethereum v0.4.0, would like to keep it the same